Frends.Excel.Parse/Frends.Excel.Parse/Excel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.ComponentModel;
+using System.Text;
+using ExcelDataReader;
+using Frends.Excel.Parse.Definitions;
+
+namespace Frends.Excel.Parse;
+
+public static class Excel
+{
+    /// <summary>
+    /// Converts Excel file to data set. [Documentation](https://github.com/FrendsPlatform/Frends.Excel/tree/main/Frends.Excel.Parse)
+    /// </summary>
+    /// <param name="input">Input configuration</param>
+    /// <param name="options">Input options</param>
+    /// <param name="cancellationToken"></param>
+    /// <returns>Result containing the parsed Excel as data set.</returns>
```

Comment:
Maybe the return documentation should include the object with property names, so users can read from the documentation how they can refer to the properties of the result object. Like `Result object {string FirstProperty, int SecondProperty}`.

Reply:
Yes, this is fixed.